src/python/pants/jvm/goals/coursier.py

```diff
@@ -67,14 +66,8 @@ async def map_resolves_to_consuming_targets(
     for tgt in all_targets:
         if not tgt.has_field(JvmArtifactCompatibleResolvesField):
             continue
-        # TODO: add a `default_compatible_resolves` field.
-        # TODO: error if no resolves for the target. Wait to change until we automatically set a
-        #  default resolve.
```

Comment:
Given that we a) decided to not have `[jvm].default_compatible_resolves` in https://github.com/pantsbuild/pants/pull/13902, and b) we plan to set a default for `default_resolve`, this won't be able to happen soon. There will always be a resolve defined.